### ISSUE TITLE
Cordova-cli 9 support

### ISSUE THREAD

**Issue by ParameshSS** [becvert/cordova-plugin-zeroconf#74]
Hello Sir,

I am using this plugin in my application, I have updated cordova to latest version(9.0.0) but I am getting error like Using "requireCordovaModule" to load non-cordova module "glob" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.

Does this plugin supported to latest version of cordova??

please help me out to resolve this issue.

Thank you.

**Comment by becvert** [comment]
Hello,
This plugin is dependent on cordova-plugin-add-swift-support for iOS.
That plugin does not [support cordova-cli 9](https://github.com/akofman/cordova-plugin-add-swift-support/issues/56) yet.

**Comment by lrbalt** [comment]
the upgrade is being worked, see on https://github.com/akofman/cordova-plugin-add-swift-support/pull/57

**Comment by ParameshSS** [comment]
Hello
@lrbalt I want to use zeroconf, zeroconf is dependent on this plugin I tried adding swift-support plugin before adding zeroconf then also i am getting the same error.

**Comment by becvert** [comment]
The dependency is added automatically for you.
We're just waiting on swift-support plugin to be updated now.

**Comment by ParameshSS** [comment]
Thank for your reply, when can we expect the update Sir.

**Comment by becvert** [comment]
It's not up to me. The guys over there seem quite active. A few days max maybe or sooner...

**Comment by TomMettam** [comment]
+1, v9 support would be amazing :)

**Comment by becvert** [comment]
Fixed!

**Comment by lrbalt** [comment]
It works for me. It took me a while to figure out how to remove old version of the `plugin-add-swift-support`. I manually changed it, so the cordova cli-tooling did not remove it. I just deleted `plugins/plugin-add-swift-support` directory. Then, `cordova plugin rm cordova-plugin-zeroconf` worked. Adding using `cordova plugin add cordova-plugin-zeroconf` then worked. I also removed both plugins from `package.json` and `config.xml`, but I'm not sure it was necessary at all.

I then got errors about method `watch` not being found on the plugin. I solved this by rebuilding the ios project workspace: `cordova platform remove ios` and then `cordova platform add ios`. 

My app now works in the xcode simulator again.

Thanks all!
